### ISSUE TITLE
Support for dark theme qr codes

### ISSUE THREAD

**Issue by Karakurik** [DCNick3/pqrs#6]
![Screenshot_20231129_233518_Chrome.jpg](https://github.com/DCNick3/pqrs/assets/70725055/2e6d8d54-9ed0-4c1b-9f37-3a2435d815c5)

**Comment by DCNick3** [comment]
Yeah, we don't support inverted QR codes, as it is non-standard and at the same time requires basically doubling the processing time (you just invert colors and try scanning again).

This can be implemented, but it should be an option in that case

**Comment by DCNick3** [comment]
Just to establish some expectations here: I probably won't be working on this feature anytime soon, but I will gladly accept a PR that is well-coded and does not reduce performance when inverted QR codes are not needed
